deepchem/feat/dtnn_featurizer.py

```diff
@@ -0,0 +1,50 @@
+from deepchem.feat import Featurizer
+import numpy as np
+
+
+class DTNNFeaturizer(Featurizer):
```

Comment:
This is a backwards compatibility breaking change (the current TF implementation doesn't use this). Let's consider if this is really required and if so open up a separate PR for this change